uaclient/cli.py

```diff
@@ -1181,8 +1181,17 @@ def action_collect_logs(args, *, cfg: config.UAConfig):
             ),
         )
 
-        for log in ua_logs:
+        extra_logs = ("/etc/cloud/build.info",)
+        logs = ua_logs + extra_logs
+
+        for log in logs:
             if os.path.isfile(log):
+                with open(log, "r+") as f:
+                    content = f.read()
+                    if len(content) > 0:
+                        f.seek(0)
+                        f.truncate()
+                        f.write(util.redact_sensitive_logs(content))
```

Comment:
I think if we just run write directly we will already overwrite the content of the file, right ? Or I am missing something ?